.github/pull_request_template.md

```diff
@@ -0,0 +1,12 @@
+## What is being delivered?
+
+Describe here all the changes that will be made and what the expected result is. If an issue was created you should also
+reference it!
+
+## What impacts?
+
+Describe what impacts this delivery has and whether it can cause side effects in other parts of the application.
+
+## Reversal plan
+
+Describe which plan we should follow if this delivery has to be reversed.
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**question:** As we're working with a lib, do you think it makes sense to include a reversal plan?

Reply:
Isn't that a case for a patch (following the [semver](https://semver.org/) specification, mentioned in the [`CHANGELOG`](https://github.com/juntossomosmais/django-stomp/blob/master/CHANGELOG.md#changelog))?

But, let me try to rephrase my question. Will each PR have a different reversal plan?